framework/packages/abstract-std/src/account.rs

```diff
@@ -104,7 +104,11 @@ pub mod state {
 }
 
 #[cosmwasm_schema::cw_serde]
-pub struct MigrateMsg {}
+pub struct MigrateMsg {
+    /// This field provides the new code id of the contract
+    /// This is only necessary for migrations from XION accounts.
+    pub code_id: Option<u64>,
+}
```

Comment:
Won't it be simpler to have it non-optional?

Reply:
Would it ?

Reply:
Not really, because in a lot of places, the code id is not easily available inside our code. Plus having a field with information that is not used in the contract (for abstract->abstract migration), feels very weird